cg/services/post_processing/pacbio/metrics_parser/models.py

```diff
@@ -137,6 +140,16 @@ def extract_plate(cls, data: Any):
                     data["plate"] = match.group(1)
         return data
 
+    @model_validator(mode="before")
+    @classmethod
+    def set_sequencing_completed_at(cls, data: Any):
+        if isinstance(data, dict):
+            movie_name = data.get(SmrtLinkDatabasesIDs.MOVIE_NAME)
+            if movie_name:
+                date: str = movie_name.split("_")[1] + movie_name.split("_")[2]
+                data["run_started_at"] = datetime.strptime(date, "%y%m%d%H%M%S")
```

Comment:
Are there any time zone considerations to be had hare?

Reply:
Agreed to consider timezones when it is fixed in the PacBio server